scripts/lint-copy-mod-doc.py

```diff
@@ -22,9 +23,14 @@
             exceptions += [(ERR_MOD, fn)]
         if errno == "ERR_LIN":
             exceptions += [(ERR_LIN, fn)]
+        if errno == "ERR_SAV":
+            exceptions += [(ERR_SAV, fn)]
 
 new_exceptions = False
 
+def small_alpha_vrachy_check(lines, fn):
+    return [ (ERR_SAV, line_nr, fn) for line_nr, line in enumerate(lines) if 'ᾰ' in line ]
```

Comment:
Maybe we should exclude comments, e.g. if `ᾰ` is part of a future contributor's name? The character does seem to be used in Romanian names.

Reply:
I guess that makes sense. Another option is to change the character in a follow-up release of Lean. (Gabriel promised that we were allowed to repaint the bikeshed at a later point in time.)
But adding some comment-detection to the linter shouldn't be too hard either. Not sure if it should be in this PR though.

Reply:
@bryangingechen I believe `ᾰ` is only used in ancient Greek.  You might be mistaking it for the visually similar `ă` character which indeed exists in modern Romanian.

We can change the character at any time.  Only the initial migration takes effort.

Reply:
Ah, right! I missed that it was an alpha.